x-pack/legacy/plugins/siem/public/components/header_page/header_page.tsx

```diff
@@ -19,23 +23,59 @@ const Header = styled.header`
 
 Header.displayName = 'Header';
 
+interface DraggableArguments {
+  field: string;
+  value: string;
+}
+
 export interface HeaderPageProps {
+  'data-test-subj'?: string;
```

Comment:
This looks like a dead variable? I don't see it being used anywhere in this file.

Can we just remove this?